protocol/lavasession/consumer_types.go

```diff
@@ -213,10 +212,7 @@ func (cswp *ConsumerSessionsWithProvider) getConsumerSessionInstanceFromEndpoint
 		return nil, 0, MaximumNumberOfSessionsExceededError
 	}
 
-	randomSessionId := int64(0)
-	for randomSessionId == 0 { // we don't allow 0
-		randomSessionId = rand.Int63()
-	}
+	randomSessionId := utils.SafeRand()
```

Comment:
we are using rand here to make sessions statistically unique between unsynced parties, and not random, the generating party (consumer) has an interest for them not to collide to save up on probing requests to the provider
there is no damage if it is predictable or craftable by either party, this do however need to be as performant as we can, so let's continue with whatever generator is faster